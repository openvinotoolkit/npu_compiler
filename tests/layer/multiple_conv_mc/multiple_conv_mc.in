version https://git-lfs.github.com/spec/v1
oid sha256:a58e9a149f90feabec17ed4ab456e16130033183b86f0aa2cfe0fd1ab00af61e
size 1003520
