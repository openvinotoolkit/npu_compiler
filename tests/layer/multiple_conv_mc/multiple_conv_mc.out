version https://git-lfs.github.com/spec/v1
oid sha256:ca98bf8818b1f91a6972067e6ae4555086aa35b4a72b5808080644d01f8fd310
size 12800
